Cargo.toml

```diff
@@ -43,6 +43,33 @@ library = [
 ]
 serde = ["dep:serde", "extended-isolation-forest/serde"]
 integration-tests = []
+# Use to access the Symphonia-based alternative decoder
```

Comment:
Really like how you separated the decoders in their own features :)

Reply:
Yeah, it's what rodio does and I wasn't sure which file types were supported by bliss, this approach gives users the flexibility to choose which formats they want supported